app/src/components/combobox/ComboBox.tsx

```diff
@@ -92,7 +95,7 @@ export function ComboBox<T extends object>({
       {description && <Text slot="description">{description}</Text>}
       <FieldError>{errorMessage}</FieldError>
       <Popover css={comboBoxPopoverCSS} UNSTABLE_portalContainer={container}>
-        <ListBox>{children}</ListBox>
+        <ListBox renderEmptyState={renderEmptyState}>{children}</ListBox>
```

Comment:
I guess a slight argument for composability?

Reply:
This is one of the props provided by react-aria. I can't seem to get it working though

https://react-spectrum.adobe.com/react-aria/ComboBox.html#listbox